tests/unittests/execution_store_repo_tests.py

```
@@ -104,6 +104,30 @@ def test_ttl_implied_policy(self):
         key_doc = self.repo.get_key(store_name, key)
         self.assertEqual(key_doc.policy, "evict")
 
+    def test_update_policy(self):
```

Ruff (0.8.2)

Redefinition of unused `test_update_policy` from line 83

(F811)